### ISSUE TITLE
AYN Loki Max: SD card inacessible in Game mode but fully accessible in Desktop mode

### ISSUE THREAD

**Issue by poplarhill** [ChimeraOS/chimeraos#878]
I have just done a fresh install of ChimeraOS (version 45, build 1e44050, downloaded from chimeraos.org and transferred to USB via balenaEtcher) on an AYN Loki Max. There were no noticeable problems with the installation that I could tell. After the installation I can access the SD card (Lexar, 1tb) fine -- reading, writing, formatting, etc -- when in Desktop mode. However, when I switch to Game mode, the unit no longer sees the card. The regular Format SD Card button is grayed out, and if I try to Force Format it in the Developer options, I am told that no microSD card is found. Is this a known issue for the Loki Max? Is ther a work-around?

**Comment by pastaq** [comment]
What do the commands `lsblk` and  `df -h` report?

**Comment by poplarhill** [comment]
lsblk gives:
NAME        MAJ:MIN RM   SIZE RO TYPE MOUNTPOINTS
sda           8:0    1 928.3G  0 disk /run/media/gamer/7a83dba3-c0ea-47c2-8166-eda3ada802ae
nvme0n1     259:0    0   1.9T  0 disk 
├─nvme0n1p1 259:1    0   511M  0 part /frzr_root/boot
│                                     /boot
└─nvme0n1p2 259:2    0   1.9T  0 part /var
                                      /home
                                      /frzr_root
                                      /

df -h gives:
Filesystem      Size  Used Avail Use% Mounted on
dev             6.2G     0  6.2G   0% /dev
run             6.3G  1.9M  6.3G   1% /run
efivarfs        128K   21K  103K  17% /sys/firmware/efi/efivars
/dev/nvme0n1p2  1.9T  556G  1.4T  30% /
/dev/nvme0n1p2  1.9T  556G  1.4T  30% /frzr_root
overlay         1.9T  556G  1.4T  30% /etc
tmpfs           6.3G   80M  6.2G   2% /dev/shm
/dev/nvme0n1p2  1.9T  556G  1.4T  30% /home
/dev/nvme0n1p2  1.9T  556G  1.4T  30% /var
/dev/nvme0n1p1  511M   87M  425M  17% /boot
tmpfs           6.3G   20M  6.3G   1% /tmp
tmpfs           1.3G  260K  1.3G   1% /run/user/1000
/dev/sda        913G  2.1M  867G   1% /run/media/gamer/7a83dba3-c0ea-47c2-8166-eda3ada802ae

**Comment by pastaq** [comment]
hmm, there should be an mmcblk device present typically.

What is your 1Tb sda drive?

**Comment by poplarhill** [comment]
This appears to have mostly fixed it. The formatting went well in the Chimera app, creating /dev/sda1. When I went back to Game mode, the drive could be seen in the setting for Storage, and I could move a Steam game to it and play the game from the SD card. However, in Game mode I still can't access the Format or Force Format options -- any idea why that may be? 

Also EmuDeck refuses to allow the SD card to be chosen for it's Emulation folder, coughing up the error, "getvsyncparametersifavailable() failed for 1 times" -- I know that's an EmuDeck issue, and I don't expect any support on that here. I only mention it in case the error code helps identify the reason behind the other issues I raised.

In any case, thank you for your help!! Cheers!!
